examples/src/test/resources/arquillian.xml

```diff
@@ -16,6 +16,26 @@
     <property name="persistenceEventsActivated">true</property>
   </extension>
 
+  <extension qualifier="cucumber-report">

```

Comment:
We start to get real reporting, I think we should add a second surefire execution to our example (integration test actually ATM) to check the generated doc. WDYT?